src/components/deck/deck.js

```diff
@@ -26,7 +26,7 @@ import { useAutoPlay } from '../../utils/use-auto-play';
 
 export const DeckContext = createContext();
 const noop = () => {};
-const DEFAULT_PRINT_SCALE = 0.773;
+const DEFAULT_PRINT_SCALE = 0.959;
```

Comment:
The PDF DPI is `96`. We want to scale the slide down because it's a 1:1 px to 1/100th of an inch. However there are some unchangeable margins that make `0.96` too big. So we use `0.959` to prevent overflow.

Reply:
Can we add that above in a code comment?